pkg/models/protocoldiscovery.go

```diff
@@ -16,5 +19,13 @@ type ProtocolDiscovery interface {
 	// discovery, which could result in dynamic device profile creation.
 	//
 	// TODO: add models.ScanList (or define locally) for devices
```

Comment:
You can remove this comment, as this implementation uses a channel for passing back devices.